lib/urlwatch/jobs.py

```diff
@@ -182,18 +181,24 @@ class UrlJob(Job):
     __kind__ = 'url'
 
     __required__ = ('url',)
-    __optional__ = ('cookies', 'data', 'method', 'ssl_no_verify')
+    __optional__ = ('cookies', 'data', 'method', 'ssl_no_verify', 'http_proxy', 'https_proxy')
 
     CHARSET_RE = re.compile('text/(html|plain); charset=([^;]*)')
 
     def get_location(self):
         return self.url
 
     def retrieve(self, job_state):
+
         headers = {
             'User-agent': urlwatch.__user_agent__,
         }
 
+        proxies = {
+            'http': os.getenv('HTTP_PROXY'),
+            'https': os.getenv('HTTPS_PROXY')
```

Comment:
Add trailing `,` at the end of this line, it's easier to add new items to the dict this way in the future if needed.